lms/static/js/commerce/views/receipt_view.js

```diff
@@ -0,0 +1,198 @@
+                    currency: order.currency,
+                    purchasedDatetime: order.date_placed,
+                    totalCost: self.formatMoney(order.total_excl_tax),
+                    isRefunded: false,
+                    billedTo: {
+                        firstName: order.billing_address.first_name,
+                        lastName: order.billing_address.last_name,
+                        city: order.billing_address.city,
+                        state: order.billing_address.state,
+                        postalCode: order.billing_address.postcode,
+                        country: order.billing_address.country
+                    },
+                    items: []
+                };
+
+                receiptContext.items = _.map(
+                    order.lines,
+                    function (line) {
+                        return {
+                            lineDescription: line.description,
+                            cost: self.formatMoney(line.line_price_excl_tax)
+                        };
+                    }
+                );
+            } else {
+                receiptContext = {
+                    orderNum: order.orderNum,
+                    currency: order.currency,
+                    purchasedDatetime: order.purchase_datetime,
+                    totalCost: self.formatMoney(order.total_cost),
+                    isRefunded: order.status === "refunded",
+                    billedTo: {
+                        firstName: order.billed_to.first_name,
+                        lastName: order.billed_to.last_name,
+                        city: order.billed_to.city,
+                        state: order.billed_to.state,
+                        postalCode: order.billed_to.postal_code,
+                        country: order.billed_to.country
+                    },
+                    items: []
+                };
+
+                receiptContext.items = _.map(
+                    order.items,
+                    function (item) {
+                        return {
+                            lineDescription: item.line_desc,
+                            cost: self.formatMoney(item.line_cost)
+                        };
+                    }
+                );
+            }
+            
+            return receiptContext;
+        },
+
+        getOrderCourseKey: function (order) {
+            if (this.useEcommerceApi) {
+                var length = order.lines.length;
+                for (var i = 0; i < length; i++) {
+                    var line = order.lines[i],
+                        attribute_values = _.filter(line.product.attribute_values, function (attribute) {
+                            return attribute.name == 'course_key'
+                        });
+
+                    if (attribute_values.length > 0) {
+                        return attribute_values[0]['value'];
+                    }
+                }
+            }
+            else {
```

Comment:
nitpick: JavaScript syntax has else on the same line as the if's closing bracket i.e. 
} else {